remotesync.py

```diff
@@ -166,3 +171,123 @@ def rsync_command(self):
 
     def pessimistic_wait(self):
         time.sleep(self.LOCAL_WAIT_TIME)
+
+
+class Rclone(RemoteSyncEngine):
+    """Implements the interface for usage with Rclone"""
+    LOCAL_WAIT_TIME = 300
+    DEFAULT_FLAGS = {'RCLONE_STATS': '8h',
+                     'RCLONE_STATS_ONE_LINE': 'true',
+                     'RCLONE_LOG_FORMAT': '',
+                     'RCLONE_LOG_LEVEL': 'INFO',
+                     'RCLONE_RETRIES': '1'}
+
+    def __init__(self, **kwargs):
+        self.RCLONE_REMOTE = kwargs.pop('rclone_remote')
+        self.RCLONE_PATH = kwargs.pop('rclone_path')
+
+        # Rclone config flags can be set entirely using environment variables.
+        # This is what is done in this implementation.
+        # https://rclone.org/docs/#environment-variables
+        # Warn user that he is overriding some default flags.
+        flags = set(self.DEFAULT_FLAGS.keys()) & set(kwargs.keys())
+        if flags:
+            warning('Flags %s were passed, these might mess up log formatting, thread carefully!' % flags)
```

Comment:
Typo, I guess it should be 'tread carefully' :)

Reply:
Eagle eyes, Mr Banks. :smile: Fixed.